week-2/interpret/time-format.js

```diff
@@ -12,32 +12,32 @@ function formatTimeDisplay(seconds) {
   const totalHours = (totalMinutes - remainingMinutes) / 60;
   const remainingHours = totalHours % 24;
 
-  return `${pad(remainingHours)}:${pad(remainingMinutes)}:${pad(
-    remainingSeconds
-  )}`;
+  return `${remainingHours.toString().padStart(2, "0")}:${remainingMinutes
+    .toString()
+    .padStart(2, "0")}:${remainingSeconds.toString().padStart(2, "0")}`;
 }
 
-console.log(formatTimeDisplay(143));
+console.log(formatTimeDisplay(323485723465));
 
 // You can play computer with this example
 // Use the Python Visualiser https://pythontutor.com/visualize.html#mode=edit
 // to help you answer these questions
 
 // Questions
 
-// a) When formatTimeDisplay is called how many times will pad be called?
+// a) When formatTimeDisplay is called how many times will pad be called?                      3 times
```

Comment:
This is correct! It's been called 3 times for calculating the hours, minutes, and seconds!